src/vmware/azext_vmware/custom.py

```diff
@@ -19,7 +19,7 @@ def privatecloud_show(cmd, client: AVSClient, resource_group_name, name):
 
 
 def privatecloud_create(cmd, client: AVSClient, resource_group_name, name, location, sku, cluster_size, network_block, circuit_primary_subnet=None, circuit_secondary_subnet=None, internet=None, vcenter_password=None, nsxt_password=None, tags=[]):
-    from azext_vmware.vendored_sdks.models import PrivateCloud, Circuit, ManagementCluster, Sku
+    from azext_vmware.vendored_sdks.avs_client.models import PrivateCloud, Circuit, ManagementCluster, Sku

```

Comment:
Is this a refactor of package hierarchy?

Reply:
Not on purpose. After updating from `--use=@microsoft.azure/autorest.python@3.0.62` to `--use=@autorest/python@5.6.0`, that was the updated path.